Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.World.cpp

```diff
@@ -549,6 +549,13 @@ int CLuaFunctionDefs::SetBlurLevel(lua_State* luaVM)
     return 1;
 }
 
+int CLuaFunctionDefs::ResetBlurLevel(lua_State* luaVM)
+{
+    g_pGame->SetBlurLevel(static_cast<unsigned char>(36));
```

Comment:
```suggestion
    g_pGame->SetBlurLevel(36);
```
just 36 should just work here